packages/kbn-utils/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "target",
+    "target": "ES2019",
```

Comment:
why not `esnext` as configured in `tsconfig.base.json`?

Reply:
https://github.com/elastic/kibana/pull/76518/commits/c9eef0eb0293042b98a562a37cb8d47a519a4b1c#diff-4a05634d6bc70d5ed7c708814f7fa315R2